CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 ## 6.24.0 (Unreleased)
 
+NOTES:
+* eventarc: `google_eventarc_channel` now uses MMv1 engine instead of DCL. ([#21728](https://github.com/hashicorp/terraform-provider-google/pull/21728))
+* workbench: increased create timeout for `google_workbench_instance` to 40mins. ([#21700](https://github.com/hashicorp/terraform-provider-google/pull/21700))
```

Comment:
Should this line go to `IMPROVEMENTS`?

Reply:
meh, it could be important to note for some increased time in some failure cases.